### PR TITLE
Added support for AWS inf2 instance types

### LINKED ISSUE #5684: Cluster Autoscaler 1.25 branch Missing support for AWS inf2 instance type
Which component are you using?:

AWS autoscaler

Is your feature request designed to solve a problem? If so describe the problem this feature should solve.:

Describe the solution you'd like.:

Back port support for inf2 instance types which is supported in `master` branch.
Master branch already has that from [here](https://github.com/kubernetes/autoscaler/pull/5529)

Describe any alternative solutions you've considered.:

Not specify

Additional context.:

Not specify

### PR DESCRIPTION
**Pull request by ff-apple**
#### What type of PR is this?

🍒 from `master` | reference PR https://github.com/kubernetes/autoscaler/pull/5529

/kind feature

#### Which issue(s) this PR fixes:

Fixes #5684 

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by ff-apple** [comment]
Adding @gjtempleton here as already familiar with the change as was the reviewer of PR https://github.com/kubernetes/autoscaler/pull/5529

FYI: @mmcclean-aws

File: cluster-autoscaler/version/version.go [hunk 1]
```diff
@@ -17,4 +17,4 @@ limitations under the License.
 package version
 
 // ClusterAutoscalerVersion contains version of CA.
-const ClusterAutoscalerVersion = "1.25.1"
+const ClusterAutoscalerVersion = "1.25.2"
```
Comment (gjtempleton) [inline cluster-autoscaler/version/version.go:20]: No need to change this, we'll do it when we cut the release.

Comment (ff-apple) [reply cluster-autoscaler/version/version.go:20]: Thanks for the review. I will amend this bit.

**Review by mwielgus — APPROVED** [review]
/lgtm
/approve

**Pull request merged**